File: Golang/problems/easy/problem1672/solution.go
package solutions

// MaximumWealth function is a solution for the
// Problem #1672. Richest Customer Wealth
// from leetcode.com
func MaximumWealth(accounts [][]int) int {
	wealth := 0
	for i := 0; i < len(accounts); i++ {
		sum := 0
		for j := 0; j < len(accounts[i]); j++ {
			sum += accounts[i][j]
		}
		if sum > wealth {
			wealth = sum
		}
	}
	return wealth
}

File: C#/Problems/Easy/Problem1672/Solution.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LeetCode.Problems.Easy.Problem1672
{
    public class Solution {
        public int MaximumWealth(int[][] accounts) {
            if (accounts.Length == 0) { 
                return 0;
            }
            
            var maxWealth = 0;
            for (int i = 0;i < accounts.Length; i++){
                var total = 0;
                for (int j = 0; j < accounts[i].Length; j++) {
                    total += accounts[i][j];
                }
                if (total > maxWealth) {
                    maxWealth = total;
                }
            }
            return maxWealth;
        }
    }
}